lib/components/ToggleSwitch/index.tsx

```diff
@@ -3,12 +3,12 @@ import clsx from "clsx";
 import './ToggleSwitch.scss';
 
 interface ToggleSwitchProps {
-    onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+    onChange: (event: ChangeEvent<HTMLInputElement>) => void;
     value: boolean;
-    wrapperClassName: string;
-    className: string;
-    wrapperStyle: React.CSSProperties;
-    style: React.CSSProperties;
+    wrapperClassName?: string;
+    className?: string;
+    wrapperStyle?: React.CSSProperties;
+    style?: React.CSSProperties;
```

Comment:
if you want to take it to the next level you can always reference the Input component by:

```suggestion
    wrapperClassName?: ComponentProps<typeof Input>['wrapperClassName'];
    className?: ComponentProps<typeof Input>['className'];
    wrapperStyle?: React.CSSProperties;
    style?: React.CSSProperties;
```

Reply:
Yes I can add this one..Thanks